src/nasuta-sebastian/Fsd.Sebastian.Cs/Fsd.Sebastian.Cs1/Program.cs

```diff
@@ -6,11 +6,9 @@ class Program
     {
         static void Main(string[] args)
         {
-            HeaderAndFooter hf = new HeaderAndFooter();
-            hf.Header();
-            CharCounter cc = new CharCounter();
-            cc.Run(args);
-            hf.Footer();
+            new ContentPresenter().Header();
+            new CharCounter().Run(args);
+            new ContentPresenter().Footer();
```

Comment:
dotyczy to linii 9 i 11: absolutnie tak nie powinieneś robić - najpierw tworzysz nową instancję ContentPresenter TYLKO 1 RAZ, a później używasz jej do odwołania się do metod. Więcej o tym będę mówił na kolejnych zajęciach

Reply:
Właśnie się nad tym zastanawiałem czy to zmieniać. W zasadzie zrobiłem tak tylko dlatego, że Visual Studio mi to podpowiedział. :)